repos/system_upgrade/common/actors/ipuworkflowconfig/libraries/ipuworkflowconfig.py

```diff
@@ -92,8 +93,7 @@ def load_upgrade_paths_definitions(paths_definition_file):
     return definitions
 
 
-def load_raw_upgrade_paths_for_distro_and_flavour(distro_id, flavour, paths_definition_file='upgrade_paths.json'):
-    all_definitions = load_upgrade_paths_definitions(paths_definition_file)
+def extract_paths_upgrade_paths_for_distro_and_flavour(all_definitions, distro_id, flavour):
```

Comment:
nitpick, but why `extract_paths_upgrade_paths_for_distro_and_flavour` instead of `extract_upgrade_paths_for_distro_and_flavour`? Reading the name, it's a little bit confusing

Reply:
There is no reason, I'm just blind :). Should be fixed.